### PR TITLE
Use Promises instead of Callbacks on Debug Screen

### PR DESCRIPTION
**Pull request by mattThousand**
### Why
Better developer ergonomics.

### This Commit
This commit refactors the debug menu and the corresponding native code to use promises instead of callbacks.

Jul-14-2020 18-53-51

### CODE REVIEW

File: app/bt/nativeModule.ts [hunk 1]
```diff
@@ -117,83 +117,37 @@ export const submitDiagnosisKeys = async (
 // Debug Module
 const debugModule = NativeModules.DebugMenuModule;
 
-export const fetchDiagnosisKeys = async (
-  cb: (errorMessage: string, diagnosisKeys: ENDiagnosisKey[]) => void,
-): Promise<void> => {
-  debugModule.fetchDiagnosisKeys(cb);
+export const fetchDiagnosisKeys = async (): Promise<ENDiagnosisKey[]> => {
```
Comment (aledustet) [inline app/bt/nativeModule.ts:120]: All that red ❤️

File: ios/COVIDSafePaths.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -328,7 +328,6 @@
 		B5582D43249943DE001458A9 /* ExposureEventEmitter.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = ExposureEventEmitter.m; sourceTree = "<group>"; };
 		B576CC3D24993F4C00CDD9D9 /* Date+Extensions.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "Date+Extensions.swift"; sourceTree = "<group>"; };
 		B576CC4024993F4C00CDD9D9 /* Encodable+Extensions.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "Encodable+Extensions.swift"; sourceTree = "<group>"; };
-		B57BA02624A3CD3F00FBAA05 /* KeySubmissionModule.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = KeySubmissionModule.m; sourceTree = "<group>"; };
```
Comment (aledustet) [inline ios/COVIDSafePaths.xcodeproj/project.pbxproj:331]: this one right here is a very persistent fella

**Review by aledustet — APPROVED** [review]
🚢 :shipit:

**Review by johnschoeman — APPROVED** [review]
:shipit: 💯

**Pull request merged**